pymeasure/instruments/nf/li5660.py

```diff
@@ -0,0 +1,1665 @@
+        """ A property that allow user to set the primary PSD's R, X output common expand multiplier.
+            Display and output of expand results also requires enabling expand calculation with the
+            :CALCulate5:MATH EXP command. """,
+        validator=strict_discrete_set,
+        values=[1, 10, 100]
+    )
+
+    calculate2_multiplier = Instrument.control(
+        ":CALC2:MULT?", ":CALC5:MATH EXP; :CALC2:MULT %d",
+        """ A property that allow user to set the primary PSD's Y output common expand multiplier.
+            Display and output of expand results also requires enabling expand calculation with the
+            :CALCulate5:MATH EXP command. """,
+        validator=strict_discrete_set,
+        values=[1, 10, 100]
+    )
+
+    calculate3_multiplier = Instrument.control(
+        ":CALC3:MULT?", ":CALC5:MATH EXP; :CALC3:MULT %d",
+        """ A property that allow user to set the secondary PSD's R, X output common expand multiplier.
+            Display and output of expand results also requires enabling expand calculation with the
+            :CALCulate5:MATH EXP command. """,
+        validator=strict_discrete_set,
+        values=[1, 10, 100]
+    )
+
+    calculate4_multiplier = Instrument.control(
+        ":CALC4:MULT?", ":CALC5:MATH EXP; :CALC4:MULT %d",
+        """ A property that allow user to set the secondary PSD's Y output common expand multiplier.
+            Display and output of expand results also requires enabling expand calculation with the
+            :CALCulate5:MATH EXP command. """,
+        validator=strict_discrete_set,
+        values=[1, 10, 100]
+    )
+
+    calculate1_offset = Instrument.control(
+        "CALC1:OFFS?",":CALC1:OFFS:STAT ON; :CALC1:OFFS %g",
+        """ A property that allow user to offset for the primary PSD's X output. """,
+        validator=truncated_range,
+        values=[-105, 105]
+    )
+
+    calculate2_offset = Instrument.control(
+        "CALC2:OFFS?",":CALC2:OFFS:STAT ON; :CALC2:OFFS %g",
+        """ A property that allow user to offset for the primary PSD's Y output. """,
+        validator=truncated_range,
+        values=[-105, 105]
+    )
+
+    calculate3_offset = Instrument.control(
+        "CALC3:OFFS?",":CALC3:OFFS:STAT ON; :CALC3:OFFS %g",
+        """ A property that allow user to offset for the secondary PSD's X output. """,
+        validator=truncated_range,
+        values=[-105, 105]
+    )
+
+    calculate4_offset = Instrument.control(
+        "CALC4:OFFS?",":CALC4:OFFS:STAT ON; :CALC4:OFFS %g",
+        """ A property that allow user to offset for the secondary PSD's Y output. """,
+        validator=truncated_range,
+        values=[-105, 105]
+    )
+
+    calculate1_offset_state = Instrument.control(
+        ":CALC1:OFFS:STAT?", ":CALC1:OFFS:STAT %s",
+        """ A property that allow user to enable/disable offset for the primary PSD's X output. """,
+        validator=strict_discrete_set,
+        values={"On": 1, "Off": 0},
```

Comment:
You could also consider using `True` or `False` in stead of `'On'` and `'Off'` here (and in multiple other cases)